### PR TITLE
microfab

### PR DESCRIPTION
**Pull request by mbwhite**
Microfab is a containerized Hyperledger Fabric runtime for use in development environments

Signed-off-by: Matthew B White <whitemat@uk.ibm.com>

### CODE REVIEW

File: labs/microfab.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+---
+# Lab Name
+microfab
+
+# Short Description
+Microfab is a containerized Hyperledger Fabric runtime for use in development environments.
+
+# Scope of Lab
+'microfab' provides a single container image that allows you to quickly build, operate & govern and grow blockchain networks. It uses Hyperledger Fabric, the open source, industry standard for enterprise blockchain.
+This containerized version of Fabric can be easily configured with the selection of channels and orgs you want, and also can be started and stopped in seconds. You can interfact with it as you would any Fabric setup. Note that this uses the fabric binaries and starts Fabric with couchdb and cas for identities. It's not cut down.
```
Comment (tkuhrt) [inline labs/microfab.md:14]:
```suggestion
This containerized version of Fabric can be easily configured with the selection of channels and orgs you want, and also can be started and stopped in seconds. You can interact with it as you would any Fabric setup. Note that this uses the Fabric binaries and starts Fabric with couchdb and cas for identities. It's not cut down.
```

Comment (mbwhite) [reply labs/microfab.md]: Thanks @tkuhrt updated!!

File: labs/microfab.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+microfab
+
+# Short Description
+Microfab is a containerized Hyperledger Fabric runtime for use in development environments.
+
+# Scope of Lab
+'microfab' provides a single container image that allows you to quickly build, operate & govern and grow blockchain networks. It uses Hyperledger Fabric, the open source, industry standard for enterprise blockchain.
+This containerized version of Fabric can be easily configured with the selection of channels and orgs you want, and also can be started and stopped in seconds. You can interact with it as you would any Fabric setup. Note that this uses the Fabric binaries and starts Fabric with couchdb and CAs for identities. It's not cut down.
+
+Why another format factor of Fabric?  This one is specifically aimed at the developer for their first entry to Fabric. This was originally used withing the VSCode Blockchain Extension, and has also been used as part of the 'Full Stack Application Workshop' - seen at the 2022 Gloabl Forum.
```
Comment (denyeart) [inline labs/microfab.md:16]:
```suggestion
Why another format factor of Fabric?  This one is specifically aimed at the developer for their first entry to Fabric. This was originally used within the VSCode Blockchain Extension, and has also been used as part of the 'Full Stack Application Workshop' - seen at the 2022 Gloabl Forum.
```

**Review by jt-nti — APPROVED** [review]
microfab has been a valuable component of the vscode extension, and I've had a lot of success using it outside vscode in a Vagrant dev env, so it would be great to see it continue to develop as a Hyperledger lab project.

**Review by lehors — APPROVED** [review]
LGTM!

**Comment by mbwhite** [comment]
Hello... just wanted to check what's the next step @ryjones / @tkuhrt  :-)

**Comment by tkuhrt** [comment]
Hi, @mbwhite : From [hyperledger-labs/governance](https://github.com/hyperledger-labs/governance/issues/new/choose) select the transfer repo option. Then you should be able to transfer the repo to @ryjones to have it moved to the hyperledger-labs organization.

**Comment by mbwhite** [comment]
https://github.com/hyperledger-labs/governance/issues/15

**Pull request merged**